hw/syn/data/dc.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  tool_srcs: ["{proj_root}/hw/foundry/syn/{tool}/setup.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/start-dc.sh"
+              "{proj_root}/hw/syn/tools/{tool}/run-syn.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/constraints.sdc"]
+
+  // we rely on fusesoc to run lint for us
+  build_cmd:  "export TOP_ENTITY={dut} && mkdir -p REPORTS && ./start-dc.sh "
+  build_opts: ["-f run-syn.tcl | tee {sv_flist_gen_dir}/REPORTS/{dut}_flow.rpt"]
```

Comment:
Piping is not supported since we dont invoke a subshell with Popen.

Reply:
will solve with the solution discussed in https://github.com/lowRISC/opentitan/pull/1819#discussion_r397616003